source/MainPanel.cpp

```diff
@@ -151,17 +151,22 @@ void MainPanel::Draw()
 	
 	if(isDragging)
 	{
-		Color color(.2, 1., 0., 0.);
-		LineShader::Draw(dragSource, Point(dragSource.X(), dragPoint.Y()), .8, color);
-		LineShader::Draw(Point(dragSource.X(), dragPoint.Y()), dragPoint, .8, color);
-		LineShader::Draw(dragPoint, Point(dragPoint.X(), dragSource.Y()), .8, color);
-		LineShader::Draw(Point(dragPoint.X(), dragSource.Y()), dragSource, .8, color);
+		if(canDrag)
+		{
+			static const Color dragColor = *GameData::Colors().Get("drag select");
```

Comment:
Same thing here - maybe used `const Color &` instead.

In Escape Velocity, if you switched to an alien ship as your flagship it could completely alter the heads-up-display to look like an alien interface. You can't do that in ES yet, but it's worth writing the code in such a way that color and interface definitions could change while the game is running.